src/components/transport_manager/include/transport_manager/transport_manager_default.h

```diff
@@ -63,6 +64,18 @@ class TransportManagerDefault : public TransportManagerImpl {
    */
   virtual ~TransportManagerDefault();
 
+#if defined(BUILD_TESTS)
+  void set_ta_bluetooth(transport_adapter::TransportAdapter* ta_bluetooth);
+  void set_ta_tcp(transport_adapter::TransportAdapter* ta_tcp);
+  void set_ta_usb(transport_adapter::TransportAdapter* ta_usb);
```

Comment:
@SNiukalov I would not suggest to create such specific for test build methods. I hope that cleaning code work will be done soon. If you need to get access to this filed to test them would create them in protected mode and inherit from test.

Reply:
@ZhdanovP We need them exclusively to allow unit testing

Reply:
@ZhdanovP 
Fixed in:https://github.com/smartdevicelink/sdl_core/pull/3134/commits/f653a628ed63e5a95fa2564a8700c348b865b70b